src/components/Reports/LocationSection.tsx

```diff
@@ -122,6 +125,12 @@ export default function LocationSection(props: LocationSectionProps): JSX.Elemen
     }
   }, [dispatch, selectedOrganization, trackingV2]);
 
+  useEffect(() => {
+    if (plantingSite) {
+      dispatch(requestSiteReportedPlants(plantingSite.id));
+    }
+  }, [plantingSite]);
```

Comment:
`plantingSite.id`?

Reply:
Can you validate plantingSite?.id ? I’m probably missing something?

Reply:
Do you mean making it 
`dispatch(requestSiteReportedPlants(plantingSite?.id));`.  ?

I can't do that because I get `Argument of type 'number | undefined' is not assignable to parameter of type 'number'.
  Type 'undefined' is not assignable to type 'number'.ts(2345`

Reply:
```
 useEffect(() => {
    if (plantingSite?.id) {
      dispatch(requestSiteReportedPlants(plantingSite.id));
    }
  }, [plantingSite?.id, dispatch]);
```